content/productivity/custom-distribute-silver/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Custom Distribute Silver
+date: 2021-10-03T21:03:00
+subtitle: Custom distribute your silver!
+version: 0.0.1
```

Comment:
Its a little confusing but we use version to say which dark forest api version the plugin is targetting. So this should be 
```suggestion
version: 0.6.4
```